CIPs/CIP-7/CIP-7.md

```diff
@@ -43,7 +43,7 @@ The **genesis record** is stored in IPLD using the *dag-cbor* codec. It has the
 
 No other properties are allowed to be defined.
 
-**Signed records** of a CAIP-10 Link document are also encoded using *dag-cbor*. Other than that they conform to the standard record format. The content of the `data` property should be a *proof* as returned by the [3id-blockchain-utils](https://github.com/3box/js-3id-blockchain-utils) library. This library has a general structure for creating and verifying "link proofs" for any blockchain. 
+**Signed records** of a CAIP-10 Link stream are also encoded using *dag-cbor*. Other than that they conform to the standard record format. The content of the `data` property should be a *proof* as returned by the [3id-blockchain-utils](https://github.com/3box/js-3id-blockchain-utils) library. This library has a general structure for creating and verifying "link proofs" for any blockchain. 
```

Comment:
Should we update `record` to `commit` throughout this page while we're here?

Reply:
yes, thanks.